src/main/java/core/basesyntax/AreaCalculetion.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculetion {

```

Comment:
Don't use verbs for class/interface names. A better name would be something like `AreaCalculator` to represent the functionality in a more standard way.